app/css/bootcamp/components/editor-information-tooltip.css

```diff
@@ -75,6 +75,8 @@
     &.error {
         @apply bg-white border-2 border-red-500;
 
+        @apply p-0;
```

Comment:
Why this? Does this affect the different Jiki errors? (syntax, parse, logic, etc) Please check them.

Reply:
@dem4ron bump on this.